apis/v1alpha2/nginxproxy_types.go

```diff
@@ -297,6 +297,17 @@ type NginxLogging struct {
 	// +optional
 	// +kubebuilder:default=info
 	AgentLevel *AgentLogLevel `json:"agentLevel,omitempty"`
+
+	// LogFormats defines custom log formats that can be used in access logs.
+	// Each log format must have a unique name.
+	//
+	// +optional
+	LogFormat *LogFormat `json:"logFormat,omitempty"`
+
+	// AccessLogs defines the access log settings, including the log file path, format, and optional parameters.
+	//
+	// +optional
+	AccessLog *AccessLog `json:"accessLog,omitempty"`
```

Comment:
For consistency with naming, lets change these two to this:

```suggestion
	Format *NginxLogFormat `json:"format,omitempty"`

	// AccessLogs defines the access log settings, including the log file path, format, and optional parameters.
	//
	// +optional
	AccessLog *NginxAccessLog `json:"accessLog,omitempty"`
```